templates/party/add.html.twig

```diff
@@ -0,0 +1,56 @@
+{% extends 'layout-with-submenu.html.twig' %}
+
+{% block title %}{% trans %}Add Party{% endtrans %}{% endblock %}
+
```

Comment:
Does this need a translation domain?

Reply:
Added translation domain and made translations